recipes/cli11/all/conanfile.py

```diff
@@ -29,6 +29,7 @@ def package(self):
         cmake.configure(source_folder=self._source_subfolder)
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
```

Comment:
I looked at the logs and a `CLI11.pc` file is generated.
Can you add the following to `package_info` please?
```python
self.cpp_info.names["pkg_config"] = "CLI11"
```